pom.xml

```diff
@@ -2,81 +2,82 @@
-  <groupId>com.untamedears</groupId>
-  <artifactId>PrisonPearl</artifactId>
-  <packaging>jar</packaging>
-  <version>2.4.1</version>
-  <name>PrisonPearl</name>
-  <url>https://github.com/Civcraft/PrisonPearl</url>
-    
-  <properties>
-    <maven.compiler.source>1.6</maven.compiler.source>
-    <maven.compiler.target>1.6</maven.compiler.target>
-  </properties>
-  
-  <build>
-    <sourceDirectory>${basedir}/src</sourceDirectory>
-    
-    <resources>
-      <resource>
-        <directory>${basedir}</directory>
-        <includes>
-          <include>*.yml</include>
-          <include>LICENSE.txt</include>
-        </includes>
-      </resource>
-    </resources>
-  </build>
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.untamedears</groupId>
+	<artifactId>PrisonPearl</artifactId>
+	<packaging>jar</packaging>
+	<version>2.4.2</version>
+	<name>PrisonPearl</name>
+	<url>https://github.com/Civcraft/PrisonPearl</url>
 
-  <dependencies>
-    <dependency>
-      <groupId>org.spigotmc</groupId>
-      <artifactId>Spigot1.8.3</artifactId>
-      <version>1.8.3</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>vg.civcraft.mc.namelayer</groupId>
-      <artifactId>NameLayer</artifactId>
-      <version>2.1.1</version>
-      <scope>provided</scope>
-    </dependency>
-	<dependency>
-      <groupId>vg.civcraft.mc.mercury</groupId>
-      <artifactId>Mercury</artifactId>
-      <version>1.0.1-11</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>com.wimbli</groupId>
-      <artifactId>WorldBorder</artifactId>
-      <version>1.8.3</version>
-      <scope>provided</scope>
-    </dependency>
-	<dependency>
-      <groupId>com.trc202</groupId>
-      <artifactId>Combattag</artifactId>
-      <version>1.8.3</version>
-      <scope>provided</scope>
-    </dependency>
-	<dependency>
-      <groupId>net.minelink</groupId>
-      <artifactId>CombatTagPlus</artifactId>
-      <version>[1.2.0,)</version>
-      <scope>provided</scope>
-    </dependency>
-  </dependencies>
-  
-  <repositories>
-    <repository>
-      <id>civcraft-repo</id>
-      <url>http://civcraft.co:8080/plugin/repository/everything/</url>
-    </repository>
-	<repository>
-      <id>minelink-releases</id>
-      <url>http://repo.minelink.net/content/repositories/releases/</url>
-    </repository>
-  </repositories>
+	<properties>
+		<maven.compiler.source>1.6</maven.compiler.source>
+		<maven.compiler.target>1.6</maven.compiler.target>
+	</properties>
+
+	<build>
+		<sourceDirectory>${basedir}/src</sourceDirectory>
+
+		<resources>
+			<resource>
+				<directory>${basedir}</directory>
+				<includes>
+					<include>*.yml</include>
+					<include>LICENSE.txt</include>
+				</includes>
+				<filtering>true</filtering>
+			</resource>
+		</resources>
+	</build>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.spigotmc</groupId>
+			<artifactId>Spigot1.8.7</artifactId>
```

Comment:
Huh, we'll that's the problem. A project should make every effort not to depend on two versions of the same software. There are ways around this but, like you're correctly stating, none that will work with automated dependency resolution from a repository. 

Do you have a good idea in general where multiple versions of the same dependency are leveraged? I'd be happy to take it on as a task to eliminate that. With prejudice.